x-pack/plugins/monitoring/dev_docs/how_to/apm_tracing.md

```diff
@@ -0,0 +1,43 @@
+To get deeper information about the behavior of the stack monitoring application you can configure APM tracing.
+
+## For local development
+
+For Elasticians doing local development you can simply export these env vars:
+
+```shell
+export ELASTIC_APM_ACTIVE=true
+export ELASTIC_APM_ENVIRONMENT="dev-${USER}"
+```
+
+And data will be available on the shared APM cluster at (internal) - to be added to https://github.com/elastic/observability-dev/tree/main/docs
```

Comment:
I'll keep this as draft until I figure out where to link for the internal APM cluster kibana. I've asked on #kibana-dev-experience slack for starters.

Reply:
Found a good place for it and updated the doc.